Storage/Rust/RustRemoteTabs.swift

```diff
@@ -161,7 +161,7 @@ public class RustRemoteTabs {
     public func getClient(fxaDeviceId: String) -> Deferred<Maybe<RemoteClient?>> {
         return self.getAll().bind { result in
             guard result.failureValue == nil else {
-                return deferMaybe(result.failureValue as! String)
+                return deferMaybe(result.failureValue!)
             }
```

Comment:
`⚠️nit:`
Usually I like guards but there is force unwrap here which is again fine as you have a nil check above but I am wondering if maybe an `if let` would work best here. 

```
            if let failureVal = result.failureValue {
                return deferMaybe(failureVal)
            }
```